content/articles/understanding-shared-memory-programming-with-pthreads-and-openmp/index.md

```diff
@@ -0,0 +1,252 @@
+### Introduction
+Shared memory is used to speed up data flow between programs. Depending on the circumstance, programs may operate on one or more processors. As a result, a process may have several threads. Threads are referred to as `lightweight processes.` 

```

Comment:
It would really be helpful if we help the reader understand why we call Threads 'light-weight-processes'.